mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -212,7 +214,7 @@ public static boolean hiveCatalog(Configuration conf) {
           LOG.info("Loaded Hadoop catalog {}", catalog);
           return Optional.of(catalog);
         case HIVE:
-          catalog = HiveCatalogs.loadCatalog(conf);
+          catalog = CatalogUtil.loadCatalog(HiveCatalog.class.getName(), "hive", ImmutableMap.of(), conf);
```

Comment:
Shall we replace the "hive" literals with `CatalogUtil.ICEBERG_CATALOG_TYPE_HIVE`?

Reply:
Done.